docs/advanced/naming.md

```diff
@@ -164,16 +202,19 @@ Anime.Name.2022.S01.E01-E11.1080p.BluRay.FLAC2.0.H.265-Group
 Anime.Name.2022.S01.E12-E23.1080p.BluRay.FLAC2.0.H.265-Group
 ```
 
-### Nyaa
+## Nyaa
```

Comment:
Nyaa titles should avoid *adding* information that doesn't exist in the filename. They may present the same info differently and add aliases at the end for SEO but they should never add information that doesn't exist in the filename